### PR TITLE
feat: trigger CI build from develop branch - WPB-2859

### PR DESCRIPTION
**Pull request by netbe**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-2859" title="WPB-2859" target="_blank">Sub-taskWPB-2859</a>  Trigger a 'development' build on every develop update
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

The job to trigger Development build from develop is not set on cirrus-ci.

### Notes (Optional)

Add notification to Wire when trigger is build with job link (experimental)
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -205,8 +214,8 @@ platform :ios do
             body: trigger_task_body,
             headers: authorization_header
         )
-        
-        puts triggered_task
+        # {"data":{"trigger":{"task":{"id":"6498826324803584"}}}}
+        puts triggered_task["data"]["trigger"]["task"]["id"]
```
Comment (SzymonMrozek) [inline fastlane/Fastfile]: I think actually it might be helpful to generate the url here rather than just printing id

File: fastlane/Fastfile [hunk 2]
```diff
@@ -205,8 +214,8 @@ platform :ios do
             body: trigger_task_body,
             headers: authorization_header
         )
-        
-        puts triggered_task
+        # {"data":{"trigger":{"task":{"id":"6498826324803584"}}}}
```
Comment (SzymonMrozek) [inline fastlane/Fastfile]: comment

File: .cirrus.yml [hunk 3]
```diff
@@ -44,6 +44,12 @@ publishing_binary_task_template: &PUBLISHING_BINARY_TASK_TEMPLATE
   << : *BUNDLER_CACHE_TEMPLATE
   << : *RUN_SETUP_SCRIPT_TEMPLATE
 
+task:
+  name: Development
+  << : *PUBLISHING_BINARY_TASK_TEMPLATE
```
Comment (SzymonMrozek) [inline .cirrus.yml:49]: I think you wanted non-manual trigger here, and also make sure the branch is develop

**Review by SzymonMrozek — CHANGES_REQUESTED** [review]

**Comment by SzymonMrozek** [comment]
Also please make sure to run the cirrus task before merge to make sure it actually works

File: .cirrus.yml (hunk 3 above)
Comment (netbe) [reply .cirrus.yml:49]: that's why I put it in github action

File: fastlane/Fastfile (hunk 2 above)
Comment (netbe) [reply fastlane/Fastfile]: removed

File: .github/workflows/development.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+name: (CirrusCI) Development
+
+on:  workflow_dispatch
```
Comment (SzymonMrozek) [inline .github/workflows/development.yml]: but this workflow is also manual

Comment (netbe) [reply .github/workflows/development.yml]: both, manual and on push, I still want the option to trigger it manually

Comment (netbe) [reply .github/workflows/development.yml]: though I don't see a way to test it before merging (github action)

Comment (SzymonMrozek) [reply .github/workflows/development.yml]: but if you specify 'workflow_dispatch' then if wouldn't trigger on push to develop IMO

**Review by SzymonMrozek — APPROVED** [review]

**Review by KaterinaWire — APPROVED** [review]

**Review by MarcinR — APPROVED** [review]

**Pull request merged**